games-misc/cowsay/cowsay-3.04.ebuild

```diff
@@ -0,0 +1,52 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+DESCRIPTION="configurable talking ASCII cow (and other characters)"
+HOMEPAGE="https://github.com/tnalpgge/rank-amateur-cowsay"
+SRC_URI="https://github.com/tnalpgge/rank-amateur-${PN}/archive/${P}.tar.gz"
+
+LICENSE="GPL-3"
+SLOT="0"
+KEYWORDS="~alpha ~amd64 ~hppa ~mips ~ppc64 ~x86 ~x64-solaris"
+IUSE=""
+
+RDEPEND=">=dev-lang/perl-5"
+DEPEND="${RDEPEND}"
+PATCHES=(
+	"${FILESDIR}/${P}-tongue.patch"
+	"${FILESDIR}/${P}-mech.patch"
+	"${FILESDIR}/${P}-utf8.patch"
+	"${FILESDIR}/${P}-version.patch" )
+DOCS=(README ChangeLog)
```

Comment:
You could drop ```DOCS```, since both entries are covered by the default: https://devmanual.gentoo.org/ebuild-writing/functions/src_install/index.html

Reply:
fixed